packages/select/src/FilterMultiSelect/components/Trigger/TriggerButtonBase/TriggerButtonBase.tsx

```diff
@@ -24,11 +24,14 @@ export const TriggerButtonBase: React.VFC<TriggerButtonBaseProps> = ({
       ref={buttonRef}
       className={classNames(styles.button, classNameOverride)}
     >
-      <span>{children}</span>
-      <Icon
-        icon={menuTriggerState.isOpen ? chevronUp : chevronDown}
-        role="presentation"
-      />
+      <span className={styles.innerContentContainer}>
```

Comment:
Could we get away with not adding another container? Eg. Putting the styles on the button element instead.

Reply:
Actually yes - turns out it was just the `gap` rule. Thanks